account_invoice_pricelist/models/account_invoice_line.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Don't split this from main model and put all in the file `account_invoice.py`. See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming for explanations